src/lib/chips/chip.ts

```diff
@@ -1,17 +1,154 @@
-import { Component, ElementRef, Renderer } from '@angular/core';
+import {
+  // Classes
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  OnDestroy,
+  Renderer,
 
-@Component({
-  selector: 'md-chip, [md-chip]',
+  // Functions
+  forwardRef
+} from '@angular/core';
+
+import {MdFocusable} from '../core/a11y/list-key-manager';
+
+/*
+ * A reusable component config. Utilized by the MdBasicChip
+ * as well as the child class MdChip.
+ */
+export const MD_BASIC_CHIP_COMPONENT_CONFIG: Component = {
+  selector: 'md-basic-chip, [md-basic-chip]',
   template: `<ng-content></ng-content>`,
   host: {
     // Properties
-    'class': 'md-chip',
     'tabindex': '-1',
-    'role': 'option'
-  }
-})
-export class MdChip {
+    'role': 'option',
+
+    // Attributes
+    '[attr.disabled]': 'disabled',
+    '[attr.aria-disabled]': 'isAriaDisabled',
+
+    // Events
+    '(click)': 'click($event)',
+  },
+  inputs: ['disabled'],
+  outputs: ['didfocus', 'destroy']
```

Comment:
Been digging into the code trying to remove this, but since the `ChipListKeyManager` watches for focus events, this seems to be the best way to subscribe. Otherwise, I have to get a reference to the private HTML element and do `addEventListener()`, or setup a custom subscription method...which seems silly since we have the EventEmitter 😄